camel/models/model_factory.py

```diff
@@ -105,6 +106,11 @@ def create(
                 return model_class(
                     model_type, model_config_dict, url, api_key, token_counter
                 )
+            elif model_platform.is_ai71:
+                model_class = AI71Model
+                return model_class(
+                    model_type, model_config_dict, url, api_key, token_counter
+                )
```

Comment:
we can remove this return to make the code more clean, same as the return for vllm above